backend/tests/modules/simulation/load_tests/locustfile.py

```
@@ -0,0 +1,933 @@
+        """
+
+        weight = PROFESSOR_WEIGHT
+
+        # Use list of accounts if multiple provided, otherwise single account
+        # Always use list format for account distribution (like StudentUser)
+        login_email = PROFESSOR_EMAILS if PROFESSOR_EMAILS else []
+        login_password = PROFESSOR_PASSWORDS if PROFESSOR_PASSWORDS else []
+        wait_time = between(1, 3)
+        
+        # Store cohort data for follow-up requests
+        cohorts: list = []
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)